package.json

```diff
@@ -50,18 +50,38 @@
     "ember-open-browser": "^0.5.1",
     "ember-resolver": "^4.0.0",
     "ember-source": "~2.14.1",
-    "loader.js": "^4.2.3"
+    "loader.js": "^4.2.3",
+    "release-it": "^14.2.1",
+    "release-it-lerna-changelog": "^3.1.0"
```

Comment:
I haven't used these and was surprised the amount of lines added to `yarn.lock`. To confirm, is the number of installed packages (resulting in about 3,000 changed LOC) expected?

Are there alternative release processes that we could consider?

Reply:
I was also very surprised. I think it's because the other dependencies are so outdated. I can reroll `yarn.lock` after the update in https://github.com/ember-intl/cp-validations/pull/87

I choose `release-it` because it's the recommanded way according to `adopted-ember-addons` [guidelines](https://github.com/adopted-ember-addons/program-guidelines#release-process)

Reply:
Ok. Yeah, following the recommended guidelines sounds good to me.

@buschtoens and I like to use `yarn-deduplicate` to maintain the lockfile and avoid installing multiple versions of a package. If you haven't yet, can you try running (after installing `yarn-deduplicate`):

```sh
# Check which packages could be de-duplicated
yarn-deduplicate --list

# De-duplicate packages
yarn-deduplicate && yarn

# Commit the lockfile (if there's a change)
```

Reply:
Will do.